hadoop-mapreduce/src/main/java/org/apache/accumulo/hadoop/mapreduce/lib/partition/KeyRangePartitioner.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.accumulo.hadoopImpl.mapreduce.lib.partition;
+package org.apache.accumulo.hadoop.mapreduce.lib.partition;
```

Comment:
I think we could drop `lib` from the package name.

Reply:
Think the API of the partition classes should be reworked as well?

Reply:
> Think the API of the partition classes should be reworked as well?

I think would be nice to  create convenience static methods that help create the partition files.  I think now the javadoc just says to create a file with base64 encoded splits on each line.  This could be a follow on PR.